shells/webextension/src/devtools-background.js

```diff
@@ -35,7 +35,7 @@ function createPanel() {
   panelLoaded = false;
   panelShown = false;
   chrome.devtools.inspectedWindow.eval(
-    `!!(window.__APOLLO_DEVTOOLS_GLOBAL_HOOK__.ApolloClient);`,
+    `!!(window.__APOLLO_DEVTOOLS_GLOBAL_HOOK__.ApolloClient || window.__APOLLO_DEVTOOLS_SHOULD_DISPLAY_PANEL__);`,
```

Comment:
Would you mind adding a comment here as well to explain what  `__APOLLO_DEVTOOLS_SHOULD_DISPLAY_PANEL__` is?

Reply:
Hey, since there is no Apollo client in the window context of react native debugger (the react native code with apollo is ran in a separate worker), we can't check for __APOLLO_DEVTOOLS_GLOBAL_HOOK__. To make it explicit I add a separate flag.